patterns/stories-podcast-banner.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Title: Stories Podcast Hero
+ * Slug: twentytwentyfive/stories-podcast-hero
+ * Categories: featured, hero
+ * Keywords: podcast, hero, stories
+ */
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70","left":"var:preset|spacing|70","right":"var:preset|spacing|70"}},"color":{"background":"#F6CFF4"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide has-background" style="background-color:#F6CFF4;padding-top:var(--wp--preset--spacing--70);padding-right:var(--wp--preset--spacing--70);padding-bottom:var(--wp--preset--spacing--70);padding-left:var(--wp--preset--spacing--70)"><!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|60","left":"var:preset|spacing|60"}}}} -->
+	<div class="wp-block-columns alignwide"><!-- wp:column {"width":"40%","layout":{"type":"default"}} -->
+		<div class="wp-block-column" style="flex-basis:40%"><!-- wp:image {"className":"wp-block-image size-large is-resized"} -->
+			<figure class="wp-block-image size-large is-resized"><img src="<?php echo esc_url(get_template_directory_uri()); ?>/assets/images/image-from-rawpixel-id-14264288-jpeg.png" alt="Historical figure" /></figure>
+			<!-- /wp:image -->
+		</div>
+		<!-- /wp:column -->
+
+		<!-- wp:column {"verticalAlignment":"center","width":"60%"} -->
+		<div class="wp-block-column is-vertically-aligned-center" style="flex-basis:60%"><!-- wp:heading {"level":1,"style":{"typography":{"fontSize":"48px"}}} -->
+			<h1 class="wp-block-heading" style="font-size:48px">The Stories® Podcast</h1>
+			<!-- /wp:heading -->
+
+			<!-- wp:paragraph {"style":{"typography":{"fontSize":"22px"}}} -->
```

Comment:
This one should use the default paragraph sizing and not the custom size of 22px.

Reply:
Awesome thanks for this - I was trying to match the values in the figma file on my side they hadnt honored what was already set out as the specific font size values, I'll undo that in the updated version